resources/views/components/notifications/ticket.blade.php

```diff
@@ -0,0 +1,32 @@
+@props([
+    'trigger' => '',
+    'count' => null, //int
```

Comment:
```suggestion
    'count' => 0,
```
0 seems like a sane default, which also allows us to get rid of the type comment.